samples/Org.OData.Capabilities.V1.permissions-sample.xml

```diff
@@ -0,0 +1,303 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please move to `examples` folder
Please use two spaces for indentation

Reply:
sorry, but i don't see an examples folder in the master branch. Am i looking in the wrong place?

Reply:
Added `examples` folder in master branch